source/scripts/Background/controllers/MainController.ts

```diff
@@ -165,12 +327,19 @@ const MainController = (): IMainController => {
       newAccount.xpub
     );
 
+    const initialTxsForAccount = await getInitialSysTransactionsForAccount(
```

Comment:
Use Asset and Transactions manager to identify if it's connected on EVM or SyscoinChain